src/js/providers/providers.js

```diff
@@ -35,7 +35,7 @@ define([
             var idx = _.indexOf(this.providers, p);
             if (idx < 0) {
                 // No provider matched
-                return Number.MIN_VALUE;
+                return 0;
```

Comment:
Can we use a negative index here? This would follow the precedence of indexOf which is 0 or higher if found and less than 0 if not found